sheepdog/blueprint/routes/__init__.py

```diff
@@ -1,4 +1,5 @@
-"""
+"""sheepdog.blueprint.routes
```

Comment:
The modules should leave these titles out due to the way we have the docs generation set up:
https://uc-cdis.github.io/sheepdog/build/html/api_reference/sheepdog.blueprint.html#module-sheepdog.blueprint.routes

Reply:
Oh shoot, sorry. I did not realize!